### PR TITLE
Error tracking

### LINKED ISSUE #10: Error Tracking
Size: 5

Acceptance Criteria: 

- Application integrates a production error tracking service which notifies the team whenever an exception occurs in production. 

Resources:
- https://airbrake.io/
- Some open source examples include: 
  - https://github.com/Sharagoz/rails_exception_handler
  - http://smartinez87.github.io/exception_notification/
  - http://errbit.github.io/errbit/.

### PR DESCRIPTION
**Pull request by JaredRoth**
Add two gems and a small configuration that enable error tracking to a slack channel.
(fyi, its #lookingfor-errors)

closes #10

### CODE REVIEW

File: config/environments/production.rb [hunk 1]
```diff
@@ -77,3 +78,8 @@
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
 end
+
+Rails.application.config.middleware.use ExceptionNotification::Rack,
+  :slack => {
```
Comment (Salvi6God) [inline config/environments/production.rb]: So is this just setting up the possibility of connecting Slack with the Rails app or does this do just that already?

Comment (JaredRoth) [inline config/environments/production.rb]: This does that already. If you look at the initial PR comment, the channel (within Turing) is #lookingfor-errors.

**Comment by mattr-** [comment]
:+1: :shipit:

**Comment by AllPurposeName** [comment]
👍

**Comment by BobGu** [comment]
👍 
Next time you can squash your WIP commit :)

**Comment by theonlyrao** [comment]
Nice. It's cool there's a gem for that.

**Comment by Salvi6God** [comment]
Everything looks good here! I'll go ahead and merge.

**Pull request merged**